### DISCUSSION TITLE
Manage auth and ACLs via dashboard

### DISCUSSION THREAD

**Discussion by AndreKR** [emqx/emqx#7456] [Q&A] [answered]
So far I have been using Mosquitto but I find myself setting up a lot of clients and bridges and I have enough of fiddling with passwd and acl files and constant broker restarts, so I'm looking for a broker that has a web GUI to set up authentication, ACLs and bridge topics.

Eqmx does have a great and very informative dashboard but so far I have been unable to find where I can set up client auth, ACLs and bridge topics.

Is this feature simply not available or do I have to enable some module/plugin for this?

**Comment by zmstone** [comment, marked as answer]
Hi. @AndreKR 
I have good news for you.
v5.0.0-rc.1 has been released,
in 5.0, you'll be able to edit the acl.conf from dashboard and saved applied on the fly.
see below screenshots.
the documents are not quite ready though, but the dashboard is quite self-descriptive.

<img width="661" alt="image" src="https://user-images.githubusercontent.com/164324/160912354-8c3b001f-52c8-4b86-aec2-1dd9adb65684.png">

click the three dots on the right.
<img width="765" alt="image" src="https://user-images.githubusercontent.com/164324/160912522-d5ee5f81-e29d-41ec-96de-17d5dd0bc1f8.png">
